assets/content/static/navbar.html

```diff
@@ -152,31 +152,31 @@ <h6 class="text-primary mb-md-1">GitHub Leaderboard</h6>
             <ul class="navbar-nav align-items-lg-center ml-lg-auto">
                 <li class="nav-item">
                     <a class="nav-link nav-link-icon" href="https://www.facebook.com/sustainableeducationfoundation/"
-                       target="_blank" data-toggle="tooltip" title="" data-original-title="Like us on Facebook">
+                       target="_blank" data-toggle="tooltip" title="Facebook">
                         <i class="fab fa-facebook"></i>
                         <span class="nav-link-inner--text d-lg-none">Facebook</span>
                     </a>
                 </li>
 
                 <li class="nav-item">
                     <a class="nav-link nav-link-icon" href="https://twitter.com/goasksef" target="_blank"
-                       data-toggle="tooltip" title="" data-original-title="Follow us on Twitter">
+                       data-toggle="tooltip" title="Twitter">
                         <i class="fab fa-twitter"></i>
                         <span class="nav-link-inner--text d-lg-none">Twitter</span>
                     </a>
                 </li>
 
                 <li class="nav-item">
                     <a class="nav-link nav-link-icon" href="https://www.linkedin.com/company/sefglobal/" target="_blank"
-                       data-toggle="tooltip" title="" data-original-title="Follow us on LinkedIn">
+                       data-toggle="tooltip" title="LinkedIn">
                         <i class="fab fa-linkedin"></i>
                         <span class="nav-link-inner--text d-lg-none">LinkedIn</span>
                     </a>
                 </li>
 
                 <li class="nav-item">
                     <a class="nav-link nav-link-icon" href="https://www.instagram.com/sefglobal/" target="_blank"
-                       data-toggle="tooltip" title="" data-original-title="Follow us on Instagram">
+                       data-toggle="tooltip" title="Instagram">
```

Comment:
Let's use `Follow us on Instagram` as the title. Do this for the others as well

Reply:
So @anjula-sack, @jayasanka-sack told in the footer use only the name. So shall I change it in the footer also.

Reply:
https://github.com/sef-global/sef-site/pull/1366#discussion_r1009470793 so to me its good with only the name so what do you think @anjula-sack . What is better @anjula-sack